### ISSUE TITLE
Inserting null throws error 

### ISSUE THREAD

**Issue by erikroznbeker** [dkhenry/rethinkjava#3]
Hello Daniel!

I'm using version 0.6

If value in Map is null it can not be inserted:

example:

``` java
Map a = new HashMap();
a.put("b", null);
r.run(r.db("myDb").table("myTable").insert(a));
```

throws: 

``` java
java.lang.NullPointerException
    at com.dkhenry.RethinkDB.Datum.datum(Datum.java:41)
    at com.dkhenry.RethinkDB.RqlQuery$Datum.build(RqlQuery.java:345)
    at com.dkhenry.RethinkDB.RqlQuery.build(RqlQuery.java:319)
    at com.dkhenry.RethinkDB.RqlQuery.build(RqlQuery.java:315)
    at com.dkhenry.RethinkDB.RqlConnection.run(RqlConnection.java:60)
```

Thanks for great work!

**Comment by dkhenry** [comment]
Well that seems like an oversight on my behalf. I will fix it post hast.

**Comment by dkhenry** [comment]
Should be fixed by c8ae60e41736e00294e838a927f7f29ee365c256 I am going to wait until I have a working test case to close the ticket

**Comment by dkhenry** [comment]
This was fixed a while ago, but I forgot to close it. woops.
